### ISSUE TITLE
Overlapping UI elements

### ISSUE THREAD

**Issue by EmileCalixte** [EvoTM/EvoSC#254]
**Describe the bug**
I've set up EvoSC on a Trackmania server. I did not modify the configuration files created by default.

When connecting to the server, the interface element "Live ranking" is badly placed and passes over other elements of the interface. This problem persists during the whole match, and the widget position seems to be reloaded correctly when the next match starts.

Also, there is some text that seems to be options for administrators that overlays the chat time countdown, at the end of each match.

**Expected behavior**
The elements should not overlap

**To Reproduce**
1. Setup EvoSC with default config
2. Connect to the server
3. See bad widget placement during the current match
4. Wait for the end of the match
5. As Admin or MasterAdmin, see overlapping text on the chat timer

**Screenshots/Logs**

When I connect to the server:
![image](https://user-images.githubusercontent.com/26703184/171455554-4095b682-3f4f-465b-9265-9bf6a607b7ca.png)

After the match has ended and the next map has been loaded:
![image](https://user-images.githubusercontent.com/26703184/171455860-a849c1f5-bf4b-4e6d-b616-2251f41b5c98.png)

Overlapping text on timer during chat time, at the end of each match (only for admins):
![image](https://user-images.githubusercontent.com/26703184/171457550-3637ab05-657d-444d-bf8f-7f25b06257ab.png)

**EvoSC**
 - EvoSC-Version: 0.99.78
 - Game: TM2020

**When using git (please complete the following information):**
 - Current-Commit: 686d14609ad1b980f36ad14a143dd9e0474bf1c9

**Comment by Chris92de** [comment]
Hi, yea, this is a thing we're aware of but don't really know how to fix.
This happens for the first time our grid system is initialized, a simple /reset in chat should fix this.

**Comment by snixtho** [comment]
I can tell you what happens here as ive dealt with it myself: Manialinks are executed in parallel. Each module is it's own manialink. So when the modules sends their manialink to the clients, its sll executed in parallel. The code that sorts the ui has race conditions with the slot values due to no synchronization. This is also difficult to fix as maniascript doesn't really provide any way to synchtonize these things. The whole setup would need to be re-done from scratch, and I think it would be better to do this in EvoSC# or something.

**Comment by brakerb** [comment]
I have added some counter-measures against it. It won't fix the problem but atleast align the UI properly after joining (hopefully).
